### ISSUE TITLE
Which is the most important parameter for training?

### ISSUE THREAD

**Issue by HakanOZDABAK** [ultralytics/ultralytics#2062]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello everyone, as I said I am improving object classification for my graduate project. I search this problem but I just found about mAP50. So, last semester I took Deep learning lesson and we learnt train acc vs valid acc. If valid acc break away from train acc, we say overfitting staring that point. I am curious about that for YOLOV8. In these parameter, which is train acc or val acc? How can I understand, overfitting is starting which epoch?

### Additional

![results](https://user-images.githubusercontent.com/127786293/232343500-306378d7-53d4-421d-a7c1-4f1b5caed665.png)

**Comment by glenn-jocher** [comment]
@HakanOZDABAK for YOLOv8, we do not use train acc vs val acc to identify overfitting. Instead, we use the performance metric mean average precision (mAP) to evaluate the model's performance on unseen data. Typically, we train the model for a certain number of epochs and then evaluate its mAP on a validation set. If the mAP on the validation set begins to decrease while the mAP on the training set continues to increase, this is a sign of overfitting. Therefore, to identify overfitting, you should monitor the mAP on the validation set during training. From the image you shared, it appears that the model may be overfitting after epoch 100. For more information on using mAP to evaluate object detection models, I recommend checking out the documentation on https://docs.ultralytics.com.

**Comment by HakanOZDABAK** [comment]
Thank you so much for your answer, I understood and I want to say one more thanks  to you for coding and prepare YOLOV8 model.

**Comment by glenn-jocher** [comment]
@HakanOZDABAK you're welcome, Hakan. I'm glad I could help. Monitoring mean average precision (mAP) is the recommended approach for identifying overfitting in YOLOv8, rather than using train accuracy vs validation accuracy. To determine overfitting in YOLOv8, it's important to keep an eye on the model's mAP on the validation set during training. If the mAP on the validation set starts to decrease while the mAP on the training set continues to increase, this is a sign of overfitting. I'm happy to hear that you're interested in improving your object classification for your graduate project with YOLOv8. If you have any more questions, feel free to ask.
